pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -7196,6 +7198,13 @@
   tutorial_carnegieslime_string_platforms: "All modern browsers"
   tutorial_carnegieslime_name: "Super Slime Battle"
   tutorial_carnegieslime_string_detail_programming_languages: "blocks"
+  tutorial_nafbrand_name: "Code your Brand"
+  tutorial_nafbrand_longdescription: "In Code your Brand, you will be creating a virtual contact file to create a personalized QR code with your contact details. After you complete this activity, you will be equipped with a modern business card that you can share while you're networking with people who can help or guide you in your career or college journey."
+  tutorial_nafbrand_string_detail_grades: "Grades 9+"
+  tutorial_nafbrand_string_detail_platforms: ""
+  tutorial_nafbrand_string_detail_programming_languages: "Python"
+  tutorial_nafbrand_string_platforms: "All modern browsers, Android tablet, iPad, Windows phone, Android phone, iPhone"
```

Comment:
I think we normally split these entries across the different sections for each field?  That is, we have all the `_name` fields together, and then all the `_longdescription` fields together, etc?